src/extension/app/store/bulk.js

```diff
@@ -418,7 +418,7 @@ export class BulkStore {
           openUrlsLabel,
           copyUrlsCallback,
           copyUrlsLabel,
-          6000,
+          60000, // keep for 1 minute
```

Comment:
Though successful, we should show this toast for longer than a few seconds to give users time to click the copy or open URL buttons.